scripts/train/train.py

```diff
@@ -449,10 +451,17 @@ def main(cfg: DictConfig) -> Trainer:
     scheduler = build_scheduler(scheduler_name, scheduler_config)
 
     # Loggers
-    loggers = [
-        build_logger(str(name), logger_cfg)
-        for name, logger_cfg in logger_configs.items()
-    ] if logger_configs else None
+    loggers = None
```

Comment:
This setup isn't quite what I meant before.

I think the cleanest setup is to just copy the same if statement as in composer to conditionally make the mosaic logger

```
        if os.environ.get(MOSAICML_PLATFORM_ENV_VAR, 'false').lower() == 'true' and os.environ.get(
                MOSAICML_ACCESS_TOKEN_ENV_VAR) is not None and not any(isinstance(x, MosaicMLLogger) for x in loggers):
            log.info('Detected run on MosaicML platform. Adding MosaicMLLogger to loggers.')
            mosaicml_logger = MosaicMLLogger()
            loggers.append(mosaicml_logger)
```

and then just call `mosaicml_logger.log_metrics` with whatever info you need. Everything else should just work after that I think.